index.html

```diff
@@ -16,7 +16,80 @@
     <link rel="shortcut icon" type="image/x-icon" href="favicon.ico" />
   </head>
   <body>
-    <!-- Add your HTML markup here -->
-    <!-- Remember: Use semantic HTML tags like <header>, <main>, <nav>, <footer>, <section> etc -->
+    <header>
+      <h1>Hi, It's me Saghar</h1>
+      <p class="about">
+        This site is about my Personal life and my Academic life
+      </p>
+    </header>
+    <main>
+      <section>
+        <article>
+          <h2>About me:</h2>
+          <p class="about-me">
```

Comment:
Each of these paragraphs can be in their own `p` tag. This will give you more control over the spacing between them.